application/modules/front/views/search/index.php

```diff
@@ -149,11 +148,13 @@ function truncateHadithText($hadith)
                 'urn_language' => $urn_language,
             ));
 
-            echo "</div>"; // end actualHadithContainer
-
+            echo "<div class=clear></div>";
             if ($truncation) {
-                echo "<div class=searchmore><a href=\"$permalink\">Read more &hellip;</a></div>";
+                echo "<div class=searchmore><a href=\"$permalink\">Read more...</a></div>";
```

Comment:
I still prefer the ellipsis :)

Reply:
It can be converted back everywhere if you prefer. Though it will be tighter and less visible compared to 3 separate periods. The subtle difference can be seen in the attached image (the first one is with 3 dots, and 2nd one is with an ellipse symbol; maybe my eyes are getting weak, but the latter looks almost like an underscore to me):

![Screenshot 2021-05-16 154112](https://user-images.githubusercontent.com/8232216/118394289-37d74280-b65d-11eb-8369-40701c94bed4.png)

You can make the final decision on this.

Reply:
You can leave a space between the last word and the ellipsis symbol.